x/emissions/keeper/actor_utils/worker.go

```diff
@@ -67,7 +69,13 @@ func CloseWorkerNonce(k *keeper.Keeper, ctx sdk.Context, topicId keeper.TopicId,
 	// Get all forecasts from this topicId, nonce
 	forecasts, err := k.GetForecastsAtBlock(ctx, topicId, nonce.BlockHeight)
 	if err != nil {
-		return err
+		if errors.Is(err, collections.ErrNotFound) {
```

Comment:
should this not go in `GetForecastsAtBlock()` just like `GetInferencesAtBlock`...

Reply:
I'm good moving it in there, though for sake of all other clients of this code alredy using it like this, i reckon for now is ok. But I'm ok to change it like that and fix the other places (like i did in inferences)

Reply:
nah, quickly fixing

Reply:
fixed